src/lib/utils/sendBeacon.ts

```diff
@@ -37,7 +33,13 @@ const sendBeacon = async (manifest: IManifest): Promise<boolean> => {
 
         dbg(`Sending data via "sendBeacon" (size: ${blob.size}).`);
 
-        return nav.sendBeacon(manifest.url, blob);
+        try {
+            nav.sendBeacon(manifest.url, blob);
```

Comment:
This again would break the library because 
> The sendBeacon() method returns true if the user agent is able to successfully queue the data for transfer, Otherwise it returns false.
> https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon